cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,33 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
-
+  it('should let user to login', () => {
+    cy.login();
```

Comment:
Please move login to beforeEach and test in this case that the correct username is displayed on the page

Reply:
Thank you! Fixed.